test/update_dev_build.sql

```diff
@@ -0,0 +1,84 @@
+-- adding  accounts,  for suppliers
+
+INSERT INTO account (type_id, enterprise_id, number, parent, label)
+VALUES (2, 1, 40111002 ,0, "SUPPLIER'S ACCOUNT 1"),
+        (2, 1, 40111003 ,0, "SUPPLIER'S ACCOUNT 2");
+
+
+-- creditor group
+INSERT INTO creditor_group (enterprise_id, uuid,name, account_id, locked)
+VALUES (1,'cgsp1', 'Creditor groud supplier1', 286,0),
+        (1, 'cgsp2', 'Creditor groud supplier2', 287,0);
+
+-- creating creditors
+INSERT INTO creditor (uuid, group_uuid,text)
+VALUES ('credtSp1', HUID(BUID('cgsp1')), 'Creditor supplier 1'),
```

Comment:
The `uuid` and `group_uuid` are BINARY(16), you have to use functions describe above for that.

See comments above